src/UglyToad.PdfPig/Graphics/InlineImageBuilder.cs

```diff
@@ -11,12 +11,30 @@
     using Tokenization.Scanner;
     using Tokens;
 
-    internal class InlineImageBuilder
+    /// <summary>
+    /// InlineImageBuilder
+    /// </summary>
+    public class InlineImageBuilder
     {
+        /// <summary>
+        /// Properties
+        /// </summary>
         public IReadOnlyDictionary<NameToken, IToken> Properties { get; set; }
 
+        /// <summary>
+        /// Bytes
+        /// </summary>
         public IReadOnlyList<byte> Bytes { get; set; }
 
+        /// <summary>
+        /// CreateInlineImage
+        /// </summary>
+        /// <param name="transformationMatrix"></param>
+        /// <param name="filterProvider"></param>
+        /// <param name="tokenScanner"></param>
+        /// <param name="defaultRenderingIntent"></param>
+        /// <param name="resourceStore"></param>
+        /// <returns></returns>
         public InlineImage CreateInlineImage(TransformationMatrix transformationMatrix, IFilterProvider filterProvider,
```

Comment:
We could probably keep this method `internal` unless it's used by the other projects, it's not really something the client code will ever invoke and just makes the API noisy?

Reply:
agreed with you to not make the API noisy 😄 . I'll need to check if we really need it `public`